docs/k8s-helm.md

```diff
@@ -125,61 +193,70 @@ helm -n $NS upgrade -f values.yaml $RELEASE timeplus/timeplus-enterprise
 Please check the [release notes](/enterprise-releases) to confirm the target version of Timeplus Enterprise can be upgraded in-place, by reusing the current data and configuration. For example [2.3](/enterprise-v2.3) and [2.4](/enterprise-releases) are incompatible and you have to use migration tools.
 
 If you confirm you can upgrade to the new version, you can run the following commands to upgrade to the latest version:
+
 ```bash
 helm repo update
 helm -n $NS upgrade $RELEASE timeplus/timeplus-enterprise
 ```
 
 You can also check the available versions and upgrade to a specific version:
+
 ```bash
 helm repo update
 helm search repo timeplus -l
 helm -n $NS upgrade $RELEASE timeplus/timeplus-enterprise --version va.b.c
 ```
 
-### Uninstall and cleanup
-
-To uninstall the helm release, just run `helm -n $NS uninstall $RELEASE` to uninstall it.
+### Prometheus metrics
 
-Please note, by default, all the PVCs will not be deleted. You can use `kubectl get pvc -n $NS` and `kubectl delete pvc <pvc_name> -n $NS` to manually delete them.
+Timeplus Enterprise exposes its metrics in Prometheus format to allow monitoring the cluster status.
 
-You can run `kubectl delete namespace $NS` to delete all PVCs and the namespace.
+It is recommended to configure your existing Prometheus metrics collector such as [Grafana Agent](https://grafana.com/docs/agent/latest/) or [Vector](https://vector.dev/) to scrape metrics from timeplusd pods directly.
 
-## Operations
+The metrics of timeplusd are exposed at `:9363/metrics`. You will need to collect the metrics from all timeplusd pods. For example, if your Timeplus Enterprise is installed in `my_ns` namespace, you can configure the collector to collect metrics from
 
-### User management
```

Comment:
I don't think we need a dedicate section for user management as it is already part of UI feature. Instead we probably can have a section to mention how to deploy timeplus-cli pod and use cli. We can add this section once customer really uses timeplus-cli on k8s.

Reply:
Previous version can define users in the helm chart, now it's part of the UI/CLI. I am not 100% sure we should remove this, since it's natural for the user to look up such user management when they are checking the deployment guide and this section is to provide the link to proper docs.

Reply:
>since it's natural for the user to look up such user management when they are checking the deployment guide and this section is to provide the link to proper docs

I don't know. I think this is very personal because at least I don't feel the same way. Generally speaking, I'd feel strange if I need to manage the user differently based on different deployment (think about Grafana). Or you can think this differently, say if a customer asking "How should I manage the user", we can simply reply with "you can go to user or use timeplus cli" without knowing how they deployed.
 
Anyway, I'll leave this to you. You can add this section if necessary.